app/src/main/java/com/dessalines/thumbkey/keyboards/ESMessageEase.kt

```diff
@@ -640,13 +692,50 @@ val KB_ES_MESSAGEEASE_SHIFTED =
         ),
     )
 
+val NUM_KEY_2 = NUMERIC_KEYBOARD_MESSAGEASE.arr[0][1]
+val NUM_KEY_3 = NUMERIC_KEYBOARD_MESSAGEASE.arr[0][2]
+val KB_ES_MESSAGEEASE_NUM =
+    NUMERIC_KEYBOARD_MESSAGEASE.copy(
+        listOf(
+            listOf(
+                NUMERIC_KEYBOARD_MESSAGEASE.arr[0][0],
+                NUM_KEY_2.copy(
+                    center = NUM_KEY_2.center,
+                    swipes =
+                        NUM_KEY_2.swipes?.plus(
+                            SwipeDirection.RIGHT to
+                                KeyC(
+                                    display = KeyDisplay.TextDisplay("¡"),
+                                    action = KeyAction.CommitText("¡"),
+                                ),
+                        ),
+                ),
+                NUM_KEY_3.copy(
+                    center = NUM_KEY_3.center,
+                    swipes =
+                        NUM_KEY_3.swipes?.plus(
+                            SwipeDirection.LEFT to
+                                KeyC(
+                                    display = KeyDisplay.TextDisplay("¿"),
+                                    action = KeyAction.CommitText("¿"),
+                                ),
+                        ),
```

Comment:
also still wrong

Reply:
see my comment on the exclamation marks, same applies here